homeassistant/components/nextcloud/__init__.py

```diff
@@ -1,151 +1,28 @@
 """The Nextcloud integration."""
-from datetime import timedelta
-import logging
+from __future__ import annotations
 
-from nextcloudmonitor import NextcloudMonitor, NextcloudMonitorError
-import voluptuous as vol
-
-from homeassistant.const import (
-    CONF_PASSWORD,
-    CONF_SCAN_INTERVAL,
-    CONF_URL,
-    CONF_USERNAME,
-    Platform,
-)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import Platform
 from homeassistant.core import HomeAssistant
-from homeassistant.helpers import config_validation as cv, discovery
-from homeassistant.helpers.event import track_time_interval
-from homeassistant.helpers.typing import ConfigType
-
-_LOGGER = logging.getLogger(__name__)
-
-DOMAIN = "nextcloud"
-PLATFORMS = (Platform.SENSOR, Platform.BINARY_SENSOR)
-SCAN_INTERVAL = timedelta(seconds=60)
-
-# Validate user configuration
-CONFIG_SCHEMA = vol.Schema(
```

Comment:
We should keep the config schema for a deprecation period while we import the legacy YAML into a config flow with an import flow. See other merged PRs that added config flow for how to use an import step in the config flow and how to create a repairs issue to notify the user.